lib/fs/casefs.go

```diff
@@ -75,12 +75,7 @@ func NewCaseFilesystem(fs Filesystem) Filesystem {
 	caseFs := &caseFilesystem{
 		Filesystem: fs,
 	}
-	switch k.fstype {
-	case FilesystemTypeBasic:
-		caseFs.realCaser = newBasicRealCaser(fs)
-	default:
-		caseFs.realCaser = newDefaultRealCaser(fs)
-	}
+	caseFs.realCaser = newDefaultRealCaser(fs)
```

Comment:
move into struct initializer?

Reply:
I was thinking it wouldn't hurt to keep it as a separate interface (as it's already done), in case any future filesystem wants to implement their own thing (or for mocking in testing).

Reply:
I just mean

```
caseFs := &caseFilesystem{
    Filesystem: fs,
    realCaser: newDefaultRealCaser(fs),
}
```

Reply:
... got it now. Is there a nice English saying equivalent to the German "Tomaten auf den Augen haben" (probably not having tomatoes on your eyes) :D

Reply:
Not that I know of :D There's "har du knäck i lurarna" for the hearing equivalent...